.github/workflows/go.yml

```diff
@@ -61,6 +61,9 @@ jobs:
     - name: Check out code into the Go module directory
       uses: actions/checkout@v2
 
+    - name: Install dependency required for linux builds
+      run: sudo apt-get update && sudo apt-get install -y libudev-dev
```

Comment:
Why do you need `libudev-dev`?

Reply:
because U2F support uses libudev.

Reply:
OK! The branch is a bit out of date with master. Mind rebasing?

Reply:
see https://github.com/Versent/saml2aws/pull/951#issuecomment-1447793461 :slightly_smiling_face: